include/lsst/afw/table/Exposure.h

```diff
@@ -1,8 +1,8 @@
 // -*- lsst-c++ -*-
-/* 
+/*
```

Comment:
Fixed (on DM-1766; all my changes will go there to avoid disrupting the diffs for the ongoing reviews of other DM-1766 subtasks).